docs/book/src/providers/v1alpha2-to-v1alpha3.md

```diff
@@ -40,3 +40,7 @@
 ## The `util/restmapper` package has been removed
 
 - Controller runtime has native support for a [DynamicRESTMapper](https://github.com/kubernetes-sigs/controller-runtime/pull/554/files), which is used by default when creating a new Manager.
+
+## Generated kubeconfig secret for clusters now comes with unique per cluster admin username
```

Comment:
```suggestion
## Generated kubeconfig username changed from `admin` to `<cluster-name>-admin`
```

Reply:
The change was from `kubernetes-admin` to `<ClusterName>-admin`.